lib/connections/limiter.go

```diff
@@ -173,6 +169,14 @@ func (lim *limiter) String() string {
 	return "connections.limiter"
 }
 
+func (lim *limiter) getLimiters(remoteID protocol.DeviceID, c internalConn, isLAN bool) (io.Reader, io.Writer) {
+	lim.mu.Lock()
+	defer lim.mu.Unlock()
```

Comment:
no point using here defer as the scope is small, as this just prevents potential inlining.